Gems/Atom/RPI/Code/Source/RPI.Public/Pass/Specific/DownsampleSinglePassLuminancePass.cpp

```diff
@@ -78,31 +78,24 @@ namespace AZ::RPI
         }
         const uint32_t mipLevelCount = attachment->m_descriptor.m_image.m_mipLevels;
         RHI::AttachmentId attachmentId = attachment->GetAttachmentId();
-        const RHI::SingleDeviceImage* rhiImage = context.GetImage(attachmentId);
+        const RHI::MultiDeviceImage* rhiImage = context.GetImage(attachmentId);
         if (!rhiImage)
         {
             return;
         }
 
-        RHI::ResultCode result = RHI::ResultCode::Success;
         RHI::ImageViewDescriptor imageViewDescriptor;
         for (uint32_t mipIndex = 0; mipIndex < GetMin(mipLevelCount, SpdMipLevelCountMax); ++mipIndex)
         {
             imageViewDescriptor.m_mipSliceMin = static_cast<uint16_t>(mipIndex);
             imageViewDescriptor.m_mipSliceMax = static_cast<uint16_t>(mipIndex);
-            Ptr<RHI::SingleDeviceImageView> imageView = RHI::Factory::Get().CreateImageView();
-            result = imageView->Init(*rhiImage, imageViewDescriptor);
-            if (result != RHI::ResultCode::Success)
-            {
-                AZ_Assert(false, "DownsampleSingelPassMipChainPass failed to create RHI::SingleDeviceImageView.");
-                return;
-            }
+            Ptr<RHI::MultiDeviceImageView> imageView = const_cast<RHI::MultiDeviceImage*>(rhiImage)->BuildImageView(imageViewDescriptor);
```

Comment:
Do you still need the `Init` call?

Reply:
The getter which calls init is in `MultiDeviceShaderResourceGroupData::SetImageViewArray`.